src/components/AppController/AppController.js

```diff
@@ -58,6 +58,7 @@ var App = React.createClass({
           </Col>
           <Col lg={6}>
             <DataController
+              ensemble_name={this.state.ensemble_name}      
```

Comment:
You can simplify passing state to props using [spread attributes](https://reactjs.org/docs/jsx-in-depth.html#spread-attributes):
```
<DataController
  {...this.state}
  meta = {this.getfilteredMeta()}
/>
```
But given that definition of `this.state` is hidden in `AppMixin`, you might decide it's clearer to be explicit about the contents of the state variable you are passing. Another reason to dislike mixins, especially as used in `AppMixin`.